concent_api/concent_api/settings/base.py

```diff
@@ -209,3 +209,6 @@
 # - "Gatekeeper":  An internal helper that validates file transfer tokens.
 # - "admin-panel": Django admin panel that provides access to database content and service statistics.
 # CONCENT_FEATURES = []
+
+# Defines max length of task_id passed in Golem Messages.
+CONCENT_MAX_TASK_ID_LENGTH = 128
```

Comment:
Oh, one more thing. This should not be a setting. We don't really support user changing this arbitrarily - we need it to match golem-messages. It should be a constant. Please put it in models.